### DISCUSSION TITLE
[QUESTION] - Should it sql or sqlu for using SELECT FOR UPDATE

### DISCUSSION THREAD

**Discussion by sujit-baniya** [slick/slick#2672] [Questions]
I've a requirement to lock some row(s) for Updating/Deleting. For the purpose, I'm using Raw SQL Query as in

```sql
SELECT column FROM table_name WHERE id = ? FOR UPDATE;
```
In this case, I wouldn't want the result but only lock the specific row. I tried to use 

```sql
sqlu"""SELECT column FROM table_name WHERE id = ? FOR UPDATE;"""
```

but it returned error in slick 3.3.0 stating update statement should not return Resultset. 

Using 
```sql
sql"""SELECT column FROM table_name WHERE id = ? FOR UPDATE;""".as[AbitraryField]
``` 

doesn't sound correct.

Any suggestions for the same?

**Comment by jkugiya** [comment]
You should use `sql` with transaction.
